test/test_test.c

```diff
@@ -24,7 +24,7 @@
 
 #define C(l, b, t)                                      \
     if ((t) != b) {                                     \
-        fprintf(stderr, "FATAL : %s != %d\n", #t, b);   \
+        fprintf(stderr, "FATAL: %s != %d\n", #t, b);    \
         goto l;                                         \
```

Comment:
This goes against coding style guide. "Do not write macros that affect control flow."